### DISCUSSION TITLE
14 febbraio 2023 B - esercizio 1

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#32] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/3f1b54f9-4f6f-441f-b060-087d178215d5)

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/8b4299c1-01bf-4db2-b5ba-acb826e601f3)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/f1d425c2-5773-4941-a08e-1a0479f6844f)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/1a2526bd-6eae-4d93-80be-92bbdf6c409c)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/e16e3040-b069-410c-b4e7-3a465cdde439)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/5c28bd5e-e976-4050-9d42-d4b6f48fbe92)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/a868133b-f7d1-4771-bfbb-bea867252adb)

**Comment by IonutCicio** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/4e37b030-26ac-47c1-8c1a-f6d7326301c1)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/d09d8087-d872-464c-91a1-f4089152abcc)

**Comment by niccolo-fato** [comment]
![IMAGE 2024-01-14 15:48:36](https://github.com/sapienzastudentsnetwork/bdi2324/assets/73995334/4b32d371-48e7-46ef-a58d-34ec11ef9f1f)
